editor/src/uuiui/menu-tab.tsx

```diff
@@ -32,16 +32,15 @@ export const MenuTab: React.FunctionComponent<React.PropsWithChildren<MenuTabPro
     const label = defaultIfNull<React.ReactElement | string>('', props.label)
 
     const baseStyle = {
-      paddingLeft: 4,
-      paddingRight: 4,
-      paddingBottom: 6,
+      padding: '4px 6px',
       transition: 'all .05s ease-in-out',
       '&:hover': {
         backgroundColor: colorTheme.tabHoveredBackground.value,
         boxShadow: `inset 0px -2px 0px 0px ${colorTheme.primary.value}`,
       },
       cursor: 'pointer',
       flexGrow: 1,
+      height: '100%',
```

Comment:
We try to avoid `height: 100%` because it can have unintended consequences. To get the tabs to full height, you should instead configure the layout system that's rendering them to handle the sizing. 

In this case, that means the container row for the tabs should `stretch` them via `alignItems`.